homeassistant/components/sensor/emoncms.py

```diff
@@ -0,0 +1,246 @@
+"""
+Support for monitoring emoncms feeds.
+
+https://emoncms.org
```

Comment:
Please refer to the HASS docs here (there you can point to emoncms), similar to:

```
"""
Support for monitoring emoncms feeds.

For more details about this component, please refer to the documentation
at https://home-assistant.io/components/sensor.emoncms/
"""
```